curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md

```diff
@@ -30,7 +30,7 @@ assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1/)
 Your event listener should include a callback function for the second argument using arrow syntax without curly braces. Then you should use the `classList` property and its `toggle` method on the `taskForm`, to toggle the class `hidden`.
 
 ```js
-assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1\s*,\s*\(\)\s*\s*=>\s*taskForm\.classList\.toggle\(\s*('|"|`)hidden\2\s*\)\s*\);?/)
+assert.match(code, /openTaskFormBtn\.addEventListener\(\s*click\s*,\s*\(\)\s*\s*=>\s*taskForm\.classList\.toggle\(\s*('|"|`)hidden\2\s*\)\s*\);?/)
```

Comment:
```suggestion
assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1\s*,\s*\(\)\s*=>\s*{?\s*taskForm\.classList\.toggle\(\s*('|"|`)hidden\2\s*\)\s*}?\s*\);?/);
```

Reply:
> make the curly brackets optional and add single/double quote option for `click`

Could you add the `\s*` also here to your suggestion?
```suggestion
assert.match(code, /openTaskFormBtn\.addEventListener\(\s*click\s*,\s*\(\s*\)\s*\s*=>\s*taskForm\.classList\.toggle\(\s*('|"|`)hidden\2\s*\)\s*\);?/)
```